doc/changes/latest.inc

```diff
@@ -95,6 +97,8 @@ Enhancements
 
 Bugs
 ~~~~
+- Fix orthogonalization of power envelopes in :func:`mne.connectivity.envelope_connectivity` (:gh:`8658` **by new contributor** |Avoide|_ and `Eric Larson`_)
```

Comment:
@Avoide can you let me know what name and URL you want to have in our [`whats_new` page](https://mne.tools/dev/whats_new.html#bugs)?

Reply:
You can put in:
Qianliang Li and [URL](https://www.dtu.dk/english/service/phonebook/person?id=126774)